src/pages/Admin.tsx

```diff
@@ -12,9 +13,14 @@ const UnauthenticatedAdmin = React.lazy(
  * Component for the Admin page
  */
 const Admin: () => JSX.Element = () => {
-  const {isAuthenticated} = useAuth();
+  const {isAuthenticated, isLoading} = useAuth();
 
-  return isAuthenticated ? <AuthenticatedAdmin /> : <UnauthenticatedAdmin />;
+  if (isLoading) {
+    // TODO: Replace Loading text with loading component
```

Comment:
Is this a TODO for future or was this supposed to be for this PR? If this is for future, make a Github issue so we don't forget to come back to this

Reply:
I created an issue for this, seen [here](https://github.com/CEHAT-Clinic/dashboard/issues/199)